frontend/src/components/VPopover/meta/VPopover.stories.mdx

```diff
@@ -6,11 +6,24 @@ import {
   Story,
 } from "@storybook/addon-docs"
 
-import { placements as popoverPlacements } from "@popperjs/core"
```

Comment:
Why not use the placements from the new library here?

```
import { Placement as popoverPlacements } from "@floating-ui/dom"
```

Reply:
This imports a type, but we need a value, the list of the placement strings. Unfortunately, there's no list in the library that we could import.